lib/handler/pool.ex

```diff
@@ -10,14 +11,15 @@ defmodule Handler.Pool do
   use GenServer
 
   @type t :: %Handler.Pool{
+          delegate_fun: nil | {atom(), atom(), term()},
```

Comment:
Can you specify what each tuple elem is here? I know what it is from looking at the rest of the code. but it would be nice if the type spec would also tell. For the first one you can just use module() instead of atom() and I think will already be clearer without the need to create a new type or naming it.

Reply:
😍 that's a great point. I'll add that to the next PR